src/Microsoft.OData.Edm/Schema/EdmUntypedStructuredType.cs

```diff
@@ -0,0 +1,36 @@
+//---------------------------------------------------------------------
+// <copyright file="EdmUntypedStructuredType.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.OData.Edm
+{
+    /// <summary>
+    /// Common base class for definitions of EDM structured types.
+    /// </summary>
+    public sealed class EdmUntypedStructuredType : EdmStructuredType, IEdmStructuredType
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="EdmStructuredType"/> class.
+        /// </summary>
+        public EdmUntypedStructuredType()
+            : base(/*isAbstract*/true, /*isOpen*/true, /*baseType*/ null)

```

Comment:
Untyped is defined in the protocol as a built-in abstract type. 

---
In reply to: [127328932](https://github.com/OData/odata.net/pull/894#discussion_r127328932) [](ancestors = 127328932)